src/ace/managers/sprite.c

```diff
@@ -23,14 +23,29 @@ typedef struct tSpriteChannel {
 
 static const tView *s_pView;
 static tSpriteChannel s_pChannelsData[HARDWARE_SPRITE_CHANNEL_COUNT];
-static const tHardwareSpriteHeader CHIP s_uBlankSprite;
+static UBYTE s_ubOwnsBlankSprite;
 static tCopBlock *s_pInitialClearCopBlock;
+ULONG *s_pBlankSprite;
 
 static void spriteChannelRequestCopperUpdate(tSpriteChannel *pChannel) {
 	pChannel->ubCopperRegenCount = 2; // for front/back buffers in raw mode
 }
 
-void spriteManagerCreate(const tView *pView, UWORD uwRawCopPos) {
+void spriteManagerCreate1(const tView *pView, UWORD uwRawCopPos) {
+	s_pBlankSprite = memAllocChipClear(sizeof(ULONG));
```

Comment:
ofc if we dynamically allocate it anyway, we might as well just not allow the user to manage it and always allocate dynamically. the problem that bartman's gcc has is simply that during lto it seems to want to merge all CHIP static variables and then I cannot have a custom CHIP static variable in my program that is of a different storage type than `tHardwareSpriteHeader`.